### ISSUE TITLE
zowe config init - "The stub received bad data."

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-cli#1504]
When I run `zowe config init` I got:
```
C:\dev\node\code-review>zowe config init
Enter host (Host name of service on the mainframe.) - blank to skip:  
Enter user (User name to authenticate to service on the mainframe.) - blank to skip:
Enter password (Password to authenticate to service on the mainframe.) - blank to skip:
Unexpected Command Error:
Please review the message and stack below.
Contact the creator of handler:
"C:\dev\node\zowe-cli\node_modules\@zowe\imperative\lib\imperative\src\config\cmd\init\init.handler"
Message:
The stub received bad data.

Stack:
Error: The stub received bad data.
```

@t1m0thyj thought it was related to keytar.  So I went to windows credential manager and deleted zowe credentials.  After that, I can use `zowe config init`.  Is there some limit size of credentials I can store with the CLI?  I have ~24 projects with unique profiles + many v1 profiles.  

Current CLI version is `7.5.1`

WHAT DO

**Comment by t1m0thyj** [comment]
> WHAT DO

TL:DR - Uninstall the Secure Credential Store 🙂

Determined the probable cause of the issue is that the Secure Credential Store plug-in was still installed. We recommend uninstalling it when migrating to Zowe v2. The `zowe config convert-profiles` command automatically handles this when it migrates secure credentials.

In Zowe v1, each profile credential (user, password, etc) was stored as a separate entry in the OS vault. This was annoying on some OSes where you have to unlock the keyring once per credential (https://github.com/zowe/vscode-extension-for-zowe/issues/773).

In Zowe v2, all secure profile properties are stored in a single entry in the OS vault. There is a limitation on Windows that the maximum credential length is 2560 bytes. To overcome this limitation, if you have a lot of profiles and exceed this length, we automatically split them across multiple entries on Windows. This enhancement was never backported to the SCS plug-in so it fails to store credentials longer than 2560 bytes with this error: `The stub received bad data`.
